doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -96,14 +98,27 @@ The following list summarizes the most important changes inherited from the upst
 * Added:
 
    * Support for the Intermittently Connected Devices (ICD) Management cluster.
-   * The Kconfig options :kconfig:option:`CONFIG_CHIP_ICD_IDLE_MODE_INTERVAL`, :kconfig:option:`CONFIG_CHIP_ICD_ACTIVE_MODE_INTERVAL` and :kconfig:option:`CONFIG_CHIP_ICD_CLIENTS_PER_FABRIC` to manage ICD configuration.
+   * The Kconfig options :kconfig:option:`CONFIG_CHIP_ICD_IDLE_MODE_DURATION`, :kconfig:option:`CONFIG_CHIP_ICD_ACTIVE_MODE_DURATION` and :kconfig:option:`CONFIG_CHIP_ICD_CLIENTS_PER_FABRIC` to manage ICD configuration.
+   * New device types:
+
+     * Refidgerator
+     * Room Air Conditioner
+     * Dishwasher
+     * Laundry Washer
+     * Robotic Vacuum Cleaner
+     * Smoke CO Alarm
+     * Air Quality Sensor
+     * Air Purifier
+     * Fan
+
+   * Product Appearance attribute in the Basic Information cluster that allows to describe product color and finish.
```

Comment:
Should this be mentioned on the above page as well?

Reply:
Sorry what do you mean by the above page, as I didn't get it?

Reply:
The Data Model and device types page. Sorry, I was referring to my own comment.

Reply:
Ah ok I see. Then no, we do not mention the single attributes for the specific clusters.

Reply:
@wiba-nordic I updated the Data Model page. The devices descriptions are copy-pasted from the Matter spec, similarly to the already existing ones.